week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/shop/cart-item.component.ts

```diff
@@ -0,0 +1,21 @@
+import { div } from '../framework/dom-creators';
+import type { Item } from '../types/item';
+
+export function cartItem({ name, amount, unit, price }: Item): HTMLElement {
```

Comment:
Tutaj pamiętaj że możesz być bardziej "specyficzny" To co jest zwracane to dokładnie `HTMLDivElement` który owszem - dziedziczy po `HTMLElement` - dlatego `TypeScript`'owi to pasuje